diff --git a/src/app/reports/[gse_id]/page.tsx b/src/app/reports/[gse_id]/page.tsx
index 0b1b6bc..5aad7ed 100644
--- a/src/app/reports/[gse_id]/page.tsx
+++ b/src/app/reports/[gse_id]/page.tsx
@@ -1,8 +1,9 @@
 'use client';
 import trpc from '@/lib/trpc/client';
+import React from 'react';
 
-export default async function ReportPage({ params }: { params: Promise<{ gse_id: string }> }) {
-  const gse_id = (await params).gse_id;
+export default function ReportPage(props: { params: Promise<{ gse_id: string }> }) {
+  const params = React.use(props.params)
   //const [html, setHtml] = useState('');
 
   // useEffect(() => {
@@ -12,7 +13,7 @@ export default async function ReportPage({ params }: { params: Promise<{ gse_id:
   //     .catch(err => console.error(err));
   // }, [gse_id]); 
 
-  const { data: html = '<span>No results</span>' } = trpc.getHTML.useQuery({ id: gse_id });
+  const { data: html = '<span>No results</span>' } = trpc.getHTML.useQuery({ id: params.gse_id });
   // const fetch_html = async (id: string) => {
   //   const html = trpc.getHTML.useQuery({ id });
   //   if (html) return (html)
~